Scripts/utils.py

```diff
@@ -213,3 +214,45 @@ def print_error_title_fetch():
   print(f"  > You won't be able to delete/hide any comments like usual, but you can {F.LIGHTMAGENTA_EX}exclude users before saving the log file{S.R}")
   print(f"  > Then, you can {F.LIGHTGREEN_EX}delete the comments later{S.R} using the {F.YELLOW}mode that removes comments using a pre-existing log file{S.R}")
   input("\n Press Enter to continue...")
+
+
+'''
+  Check if the `value` is a valid option of `settings`
+  it checks:
+    - strict comparsion (prompt) eg. 'chosenVideos' == 'chosenVideos'
+    - setting index     (config file) eg. '1' == 'chosenVideos'
+    - lowercase         (config file) eg. 'chosenvideos' == 'chosenVideos'
+'''
+def getSetting( settings, value ):
```

Comment:
dont put function inside function unless necessary, it is harder to test, 

it is also apply to apply to validateInput

also maybe get_setting and validate_input to match pep8

also add type hint and docstring for the function